concent_api/conductor/service.py

```diff
@@ -54,3 +59,50 @@ def update_upload_report(file_path: str, result_transfer_request: ResultTransfer
         result_transfer_request.save()
 
         result_upload_finished.delay(result_transfer_request.subtask_id)
+
+
+def store_verification_request_and_blender_subtask_definition(
```

Comment:
is there a reason to do both things in one function? Or can it be done sequentially in two separate functions?

Reply:
For me there is no reason for this function to exist as it is used only once :)